.changeset/split-blocks.md

```diff
@@ -0,0 +1,5 @@
+---
+create-block-app: minor
```

Comment:
BTW all three templates are 0.1.0

Reply:
The reason why I preferred `0.0.x` was because we did not maintain changelog files and did not have a proper release process. Now that it's more or less sorted, we can consider switching to `0.x.x` (not necessarily now).

Reply:
I don't really mind, happy to move to `0.1.0` as long as we remember to do so for other packages as we upgrade them